datadog_checks_base/datadog_checks/base/checks/win/winpdh_base.py

```diff
@@ -13,6 +13,9 @@
     from .winpdh_stub import WinPDHCounter, DATA_TYPE_INT, DATA_TYPE_DOUBLE
 
 
+RESOURCETYPE_ANY = 0
```

Comment:
We can use `from win32netcon import RESOURCETYPE_ANY`

Reply:
I know but since it was not imported already I decided not to just in case there is a problem with that import. It's still better than a plain `0`

Reply:
👍